client/src/components/body/lab5/reading.js

```diff
@@ -0,0 +1,132 @@
+            </ul>
+
+            <h3>How to address these problems?</h3>
+            <p>WCAG, Web Content Accessibility Guidelines, includes several guidelines to improve cognitive accessibility. They define 17 specific guidelines, of which six are especially relevant for cognitive accessibility
+
+            <div className="non-bullet-list">
+                <h5>
+                    Adaptability
+                </h5>
+                <div>All information should be available in a form that can be perceived by all users. For example, the information could be spoken aloud via a narration tool. Thus you should ensure the content can be understood by the software.
+                </div>
+                <h5>
+                    Time
+                </h5>
+                <div>It is important to allow users the time they require to complete tasks. Guideline 2.2 states "provide users enough time to read and use content."
+                    People with cognitive disabilities may require more time to read content, or to perform functions such as filling out forms
+                </div>
+                <h5>
+                    Navigation
+                </h5>
+                <div>Guideline 2.4 states to include clear and descriptive headings so users can easily find information and understand relationships between different content sections</div>
+                <h5>
+                    Readability
+                </h5>
+                <div>Guideline 3.1  states "make text content readable and understandable." Keep the writing style simple and easy to understand
+                </div>
+                <h5>
+                    Predictability
+                </h5>
+                <div>Guideline 3.2 states to "make web pages appear and operate in predictable ways. Use consistency with the page layout
+                </div>
+                <h5>
+                    Input Assistance
+                </h5>
+                <div>Guideline 3.3 states to "help users avoid and correct mistakes." If they do make a mistake, ensure the message allows them to easily fix the error
+                </div>
+            </div>
+            </p>
+
+            <h3>Practical application of cognitive accessibility</h3>
+            <p>
+                We all enjoy online shopping. However, a person’s ability to use websites effectively declines by 0.8% every year over the age of 25, according to Nielsen Norman Group. Optimally designing for memory limitations will be especially important as the population ages. These techniques include:
+                <div className="non-bullet-list">
+                <h5>
+                    User Authentication
+                </h5>
+                <div>Offer at least one alternative method that does not rely on a user to memorize character strings</div>
+                <h5>
+                    Don’t hide important/frequent controls
```

Comment:
Maybe consider spacing these points, or putting them in a list to make it more readable/scannable

Reply:
Yup, done